driver/src/main/com/mongodb/rx/client/MongoClients.java

```
@@ -99,7 +107,74 @@ public static MongoClient create(final ConnectionString connectionString) {
+    }
+
+    /**
+     * Create a new client with the given connection string.
+     *
+     * <p>Note: Intended for driver and library authors to associate extra driver metadata with the connections.</p>
+     *
+     * @param connectionString the settings
+     * @param observableAdapter the {@link ObservableAdapter} to adapt all {@code Observables}.
+     * @param mongoDriverInformation any driver information to associate with the MongoClient
+     * @return the client
+     * @since 1.3
+     */
+    public static MongoClient create(final ConnectionString connectionString, final ObservableAdapter observableAdapter,
+                                     final MongoDriverInformation mongoDriverInformation) {
+        return new MongoClientImpl(com.mongodb.async.client.MongoClients.create(connectionString,
+                updateMongoDriverInformation(mongoDriverInformation)), observableAdapter);
+    }
+
+    /**
+     * Creates a new client with the given client settings.
+     *
+     * <p>Note: Intended for driver and library authors to associate extra driver metadata with the connections.</p>
+     *
+     * @param settings the settings
+     * @param observableAdapter the {@link ObservableAdapter} to adapt all {@code Observables}.
+     * @param mongoDriverInformation any driver information to associate with the MongoClient
+     * @return the client
+     * @since 1.3
+     */
+    public static MongoClient create(final MongoClientSettings settings, final ObservableAdapter observableAdapter,
+                                     final MongoDriverInformation mongoDriverInformation) {
+        return new MongoClientImpl(com.mongodb.async.client.MongoClients.create(settings,
+                updateMongoDriverInformation(mongoDriverInformation)), observableAdapter);
+    }
+
+    private static MongoDriverInformation updateMongoDriverInformation(final MongoDriverInformation mongoDriverInformation) {
```

Yeah its just to keep the code dry - as the default is used in multiple places and the updating is done in multiple places. Mind you `DEFAULT_DRIVER_INFORMATION` is slightly misleading.  But as the builder prepends the information it needs to apply the settings to the user defined settings.